janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -4023,12 +4021,36 @@ public void testTinkerPopOptimizationStrategies() {
         assertNumStep(1, 1, gts.V().has("id", numV / 5).has("weight", (numV / 5) % 5).as("x"), JanusGraphStep.class);
 
 
+        assertNumStep(superV * (numV / 5 * 2), 2, gts.V().has("id", sid).outE("knows").has("weight", P.between(1, 3)).inV().has("weight", P.between(1, 3)), JanusGraphStep.class, JanusGraphVertexStep.class);
+        assertNumStep(superV * (numV / 5 * 2), 2, gts.V().has("id", sid).outE("knows").inV().has("weight", P.between(1, 3)), JanusGraphStep.class, JanusGraphVertexStep.class);
```

Comment:
I think you can remove this and the assert above now as you have included these now already below where it's also easier to understand them as you've added comments there.

Reply:
removed them